File: 0201_0300/0293/0293.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    f.readline()
    l = list(map(lambda x, y : x * y, (int(x) for x in f.readline().split()), (int(x) for x in f.readline().split())))
    q.write(str(l.index(max(l)) + 1))

File: 0201_0300/0293/0293.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, j, m = -1, *a;

	fscanf(f, "%d", &n);
	a = (int*)malloc(sizeof(int) * n);
	for (int i = 0; i < n; ++i)
		fscanf(f, "%d", a + i);
	for (int i = 0, k; i < n; ++i)
	{
		fscanf(f, "%d", &k);
		if (k * a[i] > m)
			j = i, m = k * a[i];
	}
	fprintf(q, "%d", j + 1);
	return 0;
}
